---
bool success
string message
eagerx_dcsc_setups/MopsSensors sensors
